xonsh/pyghooks.py

```diff
@@ -1209,6 +1254,7 @@ def _xcode_style():
     globals(),
     "STYLES",
 )
+NON_PYGMENTS_RULES: tp.Dict[str, str] = {}
```

Comment:
Perhaps this should go at the top of the file?

Reply:
(the type hint is wrong too)